lib/inet_ntop.c

```diff
@@ -118,7 +118,7 @@ static char *inet_ntop6 (const unsigned char *src, char *dst, size_t size)
   for(i = 0; i < (IN6ADDRSZ / INT16SZ); i++) {
     if(words[i] == 0) {
       if(cur.base == -1)
-        cur.base = i, cur.len = 1;
+        cur.base = i; cur.len = 1;
```

Comment:
You want braces around this block too now!

Reply:
Thanks for spotting! Fixed.